src/components/featureHighlight/index.js

```diff
@@ -268,9 +296,16 @@ class FeatureHighlight extends BaseComponent {
         strokeWidth={borderWidth || defaultStrokeWidth}
         minimumRectSize={minimumRectSize}
         innerPadding={innerPadding}
+        borderRadius={borderRadius}
       >
         <TouchableWithoutFeedback style={styles.touchableOverlay} onPress={onBackgroundPress}>
+          {pageControlProps ?
+          <View flex bottom>
```

Comment:
maybe we can allow passing pageControl position to allow changing the position somehow? 
or pass `pageControlerProps.containerStyle` together with yours

Reply:
I don't see why it's necessary since it will look pretty bad somewhere else

Reply:
If someone will want to feature highlight  elements that are at the bottom of the screen? 
anyway, your call, just an option, if someone will need they will probably request it

Reply:
Yes, if someone will ask it we can change it.